sqlmesh/core/model/definition.py

```diff
@@ -1940,4 +1949,5 @@ def _refs_to_sql(values: t.Any) -> exp.Expression:
     "session_properties_": lambda value: value,
     "allow_partials": exp.convert,
     "signals": lambda values: exp.Tuple(expressions=values),
+    "default_catalog": exp.to_identifier,
```

Comment:
why do you need this? we don't show default_catalog or set it in the model right?